pkg/micropython/Makefile.dep

```diff
@@ -0,0 +1,8 @@
+USEMODULE += xtimer
+USEMODULE += stdin
+
+# MicroPython doesn't compile for <32bit platforms
+FEATURES_BLACKLIST += arch_8bit arch_16bit
+
+# This port currently requires ISR_STACK and thread_isr_stack_start
+FEATURES_BLACKLIST += arch_arm7 arch_esp32 arch_esp8266 arch_mips arch_riscv
```

Comment:
arch_esp32 and arch_esp8266  provide ISR_STACKSIZE and thread_isr_stack_start:
```
$ git grep thread_isr_stack_start | grep esp
cpu/esp32/thread_arch.c:void *thread_isr_stack_start(void)
cpu/esp8266/thread_arch.c:void *thread_isr_stack_start(void)
$ git grep ISR_STACKSIZE | grep esp
cpu/esp32/include/xtensa_conf.h:#define ISR_STACKSIZE                 (2048)
cpu/esp8266/include/xtensa_conf.h:#define ISR_STACKSIZE                 (2048)
cpu/esp_common/vendor/xtensa/portasm.S:#define configISR_STACK_SIZE    ISR_STACKSIZE
```

Reply:
the functions are guarded by DEVELHELP. I suggest fixing that in another PR, and not blocking this.

Reply:
It's usable on hifive1b with this simple fix (not sure of what the value should be):
```diff
diff --git a/cpu/fe310/include/cpu_conf.h b/cpu/fe310/include/cpu_conf.h
index 698ea7873..a32d3c01d 100644
--- a/cpu/fe310/include/cpu_conf.h
+++ b/cpu/fe310/include/cpu_conf.h
@@ -34,6 +34,8 @@
 #endif
 /** @} */
 
+#define ISR_STACKSIZE                 (2048)
+
 /**
  * @brief   Declare the heap_stats function as available
  */
```

Reply:
> It's usable on hifive1b with this simple fix (not sure of what the value should be):

nice! but the value is important here.

Reply:
> the functions are guarded by DEVELHELP. I suggest fixing that in another PR, and not blocking this.

Yeah, I don't understand why it's done this way... Only one function (`int thread_isr_stack_usage(void)`) should be guarded.

Reply:
> but the value is important here.

Indeed, I just reused the default value used for some other arch.